### ISSUE TITLE
Missing timezone

### ISSUE THREAD

**Issue by HassanHeydariNasab** [nitely/Spirit#108]
Please fix it :)

**Comment by nitely** [comment]
[It seems to be GMT +4:30](http://wwp.greenwichmeantime.com/time-zone/asia/iran/tehran/), that timezone is already there.

**Comment by HassanHeydariNasab** [comment]
In fact it is GMT+3:30 and GMT +4:30 (DST).
Afghanistan?

**Comment by nitely** [comment]
> Afghanistan?

Yeah, I actually searched for Tehran and I found it's +4:30. But these days it seems to be better to use UTC instead of GMT. I know pytz (the python dependency to manage time zones) is DST aware when you set the time in UTC but it's not when setting in GMT, so I think I better change to UTC.

The reason I didn't use UTC in the first place is coz [the generated list is quite large](https://gist.github.com/nitely/209ae55e93d0535379bf7209b92438db), I even wrote the code to [auto generate that list](https://gist.github.com/nitely/82e19dd3381886d3a01d) back then.

Also, I was not sure whether group the list by Continent or order it by time (ie: UTC +01:00, etc).
